### PR TITLE
Using xref

### LINKED ISSUE #2207: Use respec's xref feature
Right now we have a script slurping the core document to get its definitions to add links from the other specs.

It looks like we could just export the definitions and use the xref feature in respec and dump the custom code.

### PR DESCRIPTION
**Pull request by iherman**
Eventually, this will fix #2207

~However, do _NOT_ merge this yet. Currently, this branch includes only the core spec; put it out for review while I will also do a similar change for the RS spec and some notes.~

It can go to the usual comment/review round now, I have gone through most of the notes, too.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/2245.html" title="Last updated on Apr 14, 2022, 9:29 AM UTC (7fd88b8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/2245/6459c44...7fd88b8.html" title="Last updated on Apr 14, 2022, 9:29 AM UTC (7fd88b8)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
@mattgarrish I did not touch the A11y documents. If you want I can do that... tomorrow:-)

**Comment by iherman** [comment]
We use an idiom of the form:

<pre>
&lt;a href="...">&lt;code>section&lt;/code> element&lt;/a>
</pre>

with xref this is almost exactly the same code except for <code>data-lt="section"</code> in place of href. A bit convoluted and breaks the DRY principle.

Is this idiom cast in concrete?

If we took

<pre>
&lt;a>&lt;code>section&lt;\code>&lt;\a> element
</pre>

or, if we accept the shortcut:

<pre>
[^section^] element
</pre>

this could yield

<pre>
&lt;a href="...">&lt;code>section&lt;/code>&lt;\a> element
</pre>

ie, the word "element" is outside the hyperlink. I think it is worth simplifying our code...

WDYT?

@mattgarrish

**Comment by mattgarrish** [comment]
> ie, the word "element" is outside the hyperlink. I think it is worth simplifying our code.

But are we losing context for the links only to satisfy a markdown weakness?

@GeorgeKerscher @avneeshsingh Does it make a difference for reading if the link only has a name and omits whether it's an element, attribute or property? Or do readers typically get that context by reading the sentence or paragraph and then returning to the link knowing what it is?

File: epub33/rs/index.html [hunk 1]
```diff
@@ -2303,19 +2301,19 @@ <h3>Recommendations</h3>
 					operation of an EPUB Publication, particularly on platforms where the sale of an EPUB Publication
 					and the method of reading it are connected. In these cases, it is recommended that the Reading
 					System or retailer be clear about the data being collected, how it is used, and allow for user
-					opt-outs where possible. Anonymization of data is strongly recommended for the privacy and the
+					opt-outs where possible. Anonymisation of data is strongly recommended for the privacy and the
```
Comment (mattgarrish) [inline epub33/rs/index.html]: Fixing to the British spelling? 😉 
```suggestion
					opt-outs where possible. Anonymization of data is strongly recommended for the privacy and the
```

File: epub33/rs/index.html [hunk 2]
```diff
@@ -2303,19 +2301,19 @@ <h3>Recommendations</h3>
...
 					security of the user and Reading System.</p>
 
 				<p>It is also understood that user data may be required or helpful for some Reading System affordances.
-					In these cases, anonymization is strongly recommended. It is also recommended that Reading Systems
+					In these cases, anonymisation is strongly recommended. It is also recommended that Reading Systems
```
Comment (mattgarrish) [inline epub33/rs/index.html]: Same
```suggestion
					In these cases, anonymization is strongly recommended. It is also recommended that Reading Systems
```

**Comment by iherman** [comment]
> But are we losing context for the links only to satisfy a markdown weakness?

To be fair, this is not a markdown weakness. 

```html
<a><code>section<\code><\a> element
```

is also simpler, and not DRY, than what we have now. I am happy keeping to this when possible and not use `[^section^]`.

**Comment by iherman** [comment]
@mattgarrish I would propose to merge this now, and leave https://github.com/w3c/epub-specs/pull/2245#issuecomment-1097041925, conceptually, open. Ie, we can come back to this, but the current PR is consistent as is and with #2247. We can always come back to it later.

**Comment by mattgarrish** [comment]
> I would propose to merge this now, and leave [#2245 (comment)](https://github.com/w3c/epub-specs/pull/2245#issuecomment-1097041925), conceptually, open.

It's like you read my mind... 😉 

I was going to suggest the same thing because adding the linking for the a11y specification and getting rid of the terminology lists is good to asap.

**Comment by iherman** [comment]
> It's like you read my mind... 😉

That is called "mind melt" :-)

**Pull request merged**